lib/core/src/lib/viewer/components/pdf-viewer.component.html

```diff
@@ -106,6 +106,7 @@
 
         <button
             id="viewer-scale-page-button"
+            role="button" aria-pressed="true"
```

Comment:
why pressed should be set to true?

Reply:
This reset button is not act as toggle button. This button will reset that PDF size back to 100% if size has changed(by zoom in or zoom out). I set this aria-pressed="true" because screen reader will announce "button has pressed". When you re leave  focus & click again reset button it will announce again.

Reply:
check the docs: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed

```
The aria-pressed attribute indicates the current "pressed" state of a toggle button.
```

you are making this pressed state true by default for all the states, which is not correct

Reply:
You need to evaluate the state of the pressed attribute based on the actual state